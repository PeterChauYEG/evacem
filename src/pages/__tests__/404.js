import React from 'react'
import renderer from 'react-test-renderer'

// Component to be tested
import FourZeroFourPage from '../404.js'

it('renders correctly', () => {
  const tree = renderer.create(
    <FourZeroFourPage />
  )
  expect(tree).toMatchSnapshot()
})
