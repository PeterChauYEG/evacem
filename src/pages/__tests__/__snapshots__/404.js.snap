// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    NOT FOUND
  </h1>
  <p>
    You just hit a route that doesn't exist...
  </p>
</div>
`;
